app/views/trainings/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1> Treningi </h1>
+
+<ul>
+  <% @trainings.each do |training| %>
+
+    <li> <%= link_to training.name, user_training_path(current_user,training) %>     <%= "--  #{training.user.name}" %>  </li>
```

Comment:
I think we don't need these empty lines in this file, moreover it seems better for me to split this `li` tag into multiple lines for better readability